public/index.html

```diff
@@ -59,6 +59,175 @@ <h1>
         </li>
       </ul>
     </section>
+    <!-- Action: scroll section -->
+    <div class="scroll-button">
+      <span><i class="fa fa-angle-down" aria-hidden="true"></i></span>
+    </div>
+    <!-- Current Works Section -->
+    <section class="works-container">
+      <div class="header">
+        <h1>My Recent <br/>Works</h1>
+        <div id="underline"></div>
+      </div>
+      <!-- Cads 
+       1st Card
+      -->
+      <div class="card">
+        <div class="project-image"></div>
+          <div class="project-info">
+            <div class="project-name">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+          <div class="skills">
```

Comment:
- [ ] **Please `ul` tag to group all the technologies and `li` tag for each technology**

Again, as mentioned above, whenever you are trying to list things, `ul` and `li` tags will help you to achieve better accessibly and optimization.